spec/services/user_updater_spec.rb

```diff
@@ -534,14 +534,6 @@
         expect(user.user_option.skip_new_user_tips).to eq(true)
         expect(user.user_option.seen_popups).to eq([-1])
         expect(messages.map(&:data)).to contain_exactly([-1])
-
-        messages = MessageBus.track_publish('/user-tips') do
```

Comment:
Can we still add a test here? Just as a way to describe what the expected behaviour is.

Reply:
I added the test back.